libmscore/rendermidi.cpp

```diff
@@ -268,7 +268,7 @@ static void playNote(EventMap* events, const Note* note, int channel, int pitch,
                         Note* nextNote = toNote(spanner->endElement());
                         double pitchDelta = (static_cast<double>(nextNote->pitch()) - pitch) * 50.0;
                         double timeDelta = static_cast<double>(offTime - onTime);
-                        double timeStep = timeDelta / pitchDelta * 20.0;
+                        double timeStep = std::abs(timeDelta / pitchDelta * 20.0);
```

Comment:
Do we prevent 0 values of pitchDelta here?

Reply:
I'll make sure this condition is trapped in my next update to PR #7021. And for some reason, the std::abs I added here is not present in PR #7021 so I'll add it again.